README.md

```diff
@@ -20,6 +20,12 @@
 
 <p align="center">⚡ Meilisearch tools for AWS integration</p>
 
+---------
+
+⚠️ **Please note that this repository is no longer being maintained and has been deprecated. We have created this new repository [cloud-providers](https://github.com/meilisearch/cloud-providers) instead.**
```

Comment:
```suggestion
⚠️ **Please note that this repository is no longer being maintained and has been deprecated. AWS is still a provider we maintain integration with, and this is the new repository [cloud-providers](https://github.com/meilisearch/cloud-providers).**
```

Reply:
I think it is important to clearly state that we still maintain a integration with AWS :)

Reply:
Also, after merging this you can make archive this repo :)